devdocs/settingUp.md

```diff
@@ -179,7 +180,19 @@ this section.
   `runtests.bat  C:\appengine-java-sdk-1.9.27  C:\teammates -testnames component-tests,sequential-ui-tests,parallel-ui-tests`
 
 **On OS-X:**
-TBD
+* Build the project in Eclipse (`Project -> Clean`).
+* Start the dev server in Eclipse.
+* Open a Terminal window, navigate to the project folder and run the `runtests.sh` in the following manner.<br>
+  `./runtests.sh GAE_SDK_Location Project_Folder_location`<br>
+  e.g. `./runtests.sh User/someuser/appengine-java-sdk-1.9.27 User/someuser/TEAMMATES`, if the path to GAE_SDK_Location is `User/someuser/appengine-java-sdk-1.9.27` and path to Project_Folder_location is `User/someuser/TEAMMATES`.<br>
```

Comment:
Tried it and it worked :+1: need to change `User/...` to `/User/...` though.
Also to make it more Mac-like, change `User` to `Users`.